### PR TITLE
Add a Configuration class

### PR DESCRIPTION
**Pull request by jacobherrington**
This should not be a breaking change as the old behavior is still respected.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6.8
```
Comment (jacobherrington) [inline .ruby-version:1]: This got committed by accident, but we could choose to leave it here?

Comment (jonallured) [reply .ruby-version:1]: Hrm, I guess I don't feel strongly one way or the other but maybe we can bump this to a newer version if we're going to commit it?

File: README.md [hunk 2]
```diff
@@ -16,12 +17,32 @@ Or install it yourself as:
 
     $ gem install braze_ruby
 
+### Configuration
```
Comment (jonallured) [inline README.md:20]: Awesome docs, thank you so much!! 🏆

**Review by jonallured — APPROVED** [review]
This is exactly what I had in mind thanks so much for taking it on! And thanks to @feministy for this post:

http://lizabinante.com/blog/creating-a-configurable-ruby-gem/

Which outlined this awesome approach!!

File: .ruby-version (hunk 1 above)
Comment (jacobherrington) [reply .ruby-version]: I went ahead and removed it as we should be supporting a range of Ruby versions! 👍

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**